libs/sysaudio/src/webaudio.zig

```diff
@@ -114,10 +115,12 @@ pub fn requestDevice(audio: Audio, allocator: std.mem.Allocator, config: DeviceD
     }
 
     // TODO(sysaudio): introduce a descriptor type that has non-optional fields.
-    var descriptor = config;
-    descriptor.mode = descriptor.mode orelse .output;
-    descriptor.channels = descriptor.channels orelse default_channel_count;
-    descriptor.sample_rate = descriptor.sample_rate orelse default_sample_rate;
+    var descriptor = DeviceDescriptor{
```

Comment:
Please remove the TODO above this (which you fixed) :)

Reply:
Done!